### ISSUE TITLE
Events created between init-db and poll() get lost.

### ISSUE THREAD

**Issue by guettli** [shawalli/psycopg2-pgevents#15]
I want to test my code:

```
    install_trigger_function(connection, overwrite=True)
    install_trigger(connection, 'lala_logofinvoicemail', overwrite=True)
    register_event_channel(connection)

    # execute INSERT

   for evt in poll(django_connection.connection, timeout=timeout):
       assert 0, evt

    assert None, connection.notifies
```

But: `assert 0` does not happend.

Although connection.notifies contains entries.

Why not process connection.notifies always (even if select() returns ([], [], [])?

**Comment by shawalli** [comment]
Hi, thanks for opening this issue! It's been a while since I've worked on this project, so I'm re-orienting myself of the code. I did have a question about your example -- I noticed that you use `connection` to set up the trigger, but you poll using `django_connection.connection`. Was this a typo or intended? The docs state that you must use the same connection for the trigger configuration and polling.
